changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.4.0 | 2019-12-05
+
+* Add calibration data as attribute of force channels.
+* Take into account slicing of channels when returning force calibration data.
+* Fixed slicing behaviour when slicing continuous time channels within a timestep.
```

Comment:
I don't understand this one.

Reply:
I've rephrased it to make it more explicit ``Fixed bug which produced incorrect timestamps when slicing continuous channel with time values between two data points.``